build-info-extractor-maven3/src/main/java/org/jfrog/build/extractor/maven/DependencyResolutionSpy.java

```diff
@@ -0,0 +1,104 @@
+package org.jfrog.build.extractor.maven;
+
+import org.apache.commons.lang.ArrayUtils;
+import org.apache.maven.eventspy.AbstractEventSpy;
+import org.apache.maven.eventspy.EventSpy;
+import org.apache.maven.project.DependencyResolutionResult;
+import org.codehaus.plexus.component.annotations.Component;
+import org.codehaus.plexus.component.annotations.Requirement;
+import org.eclipse.aether.artifact.Artifact;
+import org.eclipse.aether.graph.DependencyNode;
+import org.jfrog.build.extractor.BuildInfoExtractorUtils;
+
+import java.util.*;
+
+/**
+ * Listen to DependencyResolutionResult events and populate parents map for each dependency of the current module.
+ * Each parent is an array of strings contains the path from parent GAV to module GAV.
```

Comment:
Each parent is an array of strings contains the path from parent GAV to module GAV.
-->
Each parent is an array of strings, containing the path from parent GAV to module GAV.